03-attribute-operations.Rmd

```diff
@@ -332,6 +333,9 @@ Most of the following join examples will have a `sf` object as the first argumen
 However, the reverse order is also possible and will give you back a `data.frame` object.
 This is mostly beyond the scope of this book, but we encourage you to try it.
 
+The next subsections will focus on the two most often used types of joins - left and inner joins.
+We recommend to read @grolemund_r_2016 to learn more about the rest of the joins types.
```

Comment:
I would delete this line and replace the preceding line with the following text to make the section flow better and more concise:

```
The next subsections focuses on the commonly used left and inner joins, which use the same syntax as the other join types [@grolemund_r_2016].
```